src/reading/parseFile.js

```diff
@@ -4,6 +4,8 @@ import mapValues from 'lodash.mapvalues'
 
 const wgs84 = gdal.SpatialReference.fromEPSG(4326)
 
+const maxLayers = 20
```

Comment:
is that a reasonable max, what happens if it is set to 100?

Reply:
The file that crashed only had like 30-something layers I believe, so I felt like 20 was a number that would work. If it's an issue, we can revisit it later.